ush/config.sh.RRFS_CONUS_13km.Ens

```diff
@@ -255,6 +266,9 @@ COMINgfs=""
 STMP="YourOwnSpace"  # Path to directory STMP that mostly contains input files.
 PTMP="YourOwnSpace"  # Path to directory STMP that mostly contains input files.
 NWGES="YourOwnSpace"  # Path to directory NWGES that save boundary, cold initial, restart files.
+STMP="/mnt/lfs4/BMC/wrfruc/Chunhua.Zhou/tmp/NCO_dirs/stmp"  # Path to directory STMP that mostly contains input files.
```

Comment:
May need to remove the 2nd set of EXPT_BASEDIR, STMP, PTMP, NWGES. But we can wait for #199 to be merged first.

Reply:
Done. Thanks!